iam-login-service/src/main/java/it/infn/mw/iam/config/IamProperties.java

```diff
@@ -232,6 +232,8 @@ public static class RegistrationProperties {
 
     Map<String, RegistrationFieldProperties> fields;
 
+    List<Map<String, String>> defaultGroups;
```

Comment:
I'd prefer to define an inner class DefaultGroup with name and enrollment fields

Reply:
it helps to define default value for enrollment (INSERT) and you can specify a NotEmpty validation for name. Of course this defaultGroups list must remain nullable as it is.

Reply:
I have added the default `INSERT` value. Do you have any example of NotEmpty validation please?

Reply:
I saw validation at this level is not done so we can live without it at this point. The existence of the group name in implicitly done later